core/src/main/java/com/linecorp/armeria/common/Flags.java

```diff
@@ -266,12 +266,30 @@
                     DEFAULT_DEFAULT_PING_INTERVAL_MILLIS,
                     value -> value >= 0);
 
+    private static final int DEFAULT_DEFAULT_MAX_SERVER_NUM_REQUESTS = 0; // Disabled
+    private static final int DEFAULT_MAX_SERVER_NUM_REQUESTS =
+            getInt("defaultMaxServerNumRequests",
+                   DEFAULT_DEFAULT_MAX_SERVER_NUM_REQUESTS,
+                   value -> value >= 0);
+
+    private static final int DEFAULT_DEFAULT_MAX_CLIENT_NUM_REQUESTS = 0; // Disabled
```

Comment:
Could be just `DEFAULT_DEFAULT_MAX_NUM_REQUESTS` and use it for client and server together?

Reply:
That was my biggest considering point. 🤔 
Originally I made only `DEFAULT_DEFAULT_MAX_NUM_REQUESTS` and split it into server and client.
Because we already have `DEFAULT_MAX_SERVER_CONNECTION_AGE_MILLIS` for only server-side.

Reply:
I think we can just use `DEFAULT_DEFAULT_MAX_CONNECTION_AGE_MILLIS`. We can change 
`DEFAULT_DEFAULT_MAX_SERVER_CONNECTION_AGE_MILLIS` to `DEFAULT_DEFAULT_MAX_CONNECTION_AGE_MILLIS` because it's private?

Reply:
Ah, I misunderstand your review. I thought you want to merge `defaultMaxClientNumRequests` and `defaultMaxServerNumRequests` into `defaultMaxNumRequests`. 🤣